Slim/Plugin/AudioScrobbler/Settings.pm

```diff
@@ -61,68 +61,128 @@ sub handler {
 
 		# Save new account
 		if ( $params->{pref_password} ) {
-			$params->{pref_password} = md5_hex( $params->{pref_password} );
+			# Last.fm Password (MD5)
+			if ( !($params->{pref_api_type} && $params->{pref_api_type} eq 'listenbrainz') ) {
+				$params->{pref_password} = md5_hex( $params->{pref_password} );
+			}
```

Comment:
The comment seems to contradict the code: is this for LastFM or for ListenBrainz?